server/server.js

```diff
@@ -12,8 +12,8 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 //api route to return all items from catalog
-app.get('/products', (req, res) => {
-  //controller.getAllItems
+app.get('/products', controllers.getAllItems(), (req, res) => {
+  res.status(200)
```

Comment:
you forgot to invoke send or json method after setting the status:
`res.status(200).send('works');`